app_dart/lib/src/request_handlers/postsubmit_luci_subscription.dart

```diff
@@ -144,12 +144,16 @@ class PostsubmitLuciSubscription extends SubscriptionHandler {
   }
 
   /// Queries the task document and updates based on the latest build data.
-  Future<void> updateFirestore(Build build, String sha, String taskName) async {
+  Future<void> updateFirestore(Build build, String commitKeyId, String taskName) async {
```

Comment:
Should we name the arg consistently (rawCommitKey)?

Reply:
From the user_data, it uses `rawCommitKey`, but this is actually the Key.id of a Commit.